composer.json

```diff
@@ -97,7 +97,7 @@
     },
     "require-dev": {
         "composer/composer": "1.0.*@dev",
-        "phpunit/phpunit": "^4.8.36"
+        "phpunit/phpunit": "^4.8.36 || ^5.7 || ^6.5"
```

Comment:
It may be worth it to leave only the latest version?

Reply:
PHPUnit 6 requires PHP 7, so, I guess it is 😅